src/tools/rust-analyzer/crates/hir-ty/src/layout.rs

```diff
@@ -173,7 +173,7 @@ pub fn layout_of_ty(db: &dyn HirDatabase, ty: &Ty, krate: CrateId) -> Result<Lay
 
             // let pointee = tcx.normalize_erasing_regions(param_env, pointee);
             // if pointee.is_sized(tcx.at(DUMMY_SP), param_env) {
-            //     return Ok(tcx.intern_layout(LayoutS::scalar(cx, data_ptr)));
+            //     return Ok(tcx.mk_layout(LayoutS::scalar(cx, data_ptr)));
             // }
```

Comment:
No idea why commented out rustc code is in r-a :laughing: 

Not sure if they want us to be touching this code, maybe revert? :shrug:

Reply:
I'm inclined to leave this in. I often do this when refactoring, better to keep the commented-out code up-to-date as much as possible, when the changes are trivial like this.